models/comment-models.js

```diff
@@ -10,4 +10,16 @@ function getCommentCountByID(article_id){
             })
 }
 
-module.exports = {getCommentCountByID}
\ No newline at end of file
+function fetchCommentsByArticleID(article_id){
+    return db.query(`SELECT *
+                     FROM comments
+                     WHERE article_id = $1
+                     ORDER BY created_at DESC`,[article_id])
+            .then(({rows})=>{
+               return rows
+            })
+            .catch((err)=>{
+            })
```

Comment:
Keep catch blocks in the controller function as if the model function fails a catch block in the controller will catch it